detd/ipc.proto

```diff
@@ -21,12 +21,13 @@ message StreamQosRequest {
 	uint32 txmax = 8;
 	bool   setup_socket = 9;
 	uint32 basetime = 10;
-	bool hints_available = 11;
-	TxSelection hints_tx_selection = 12;
-	bool hints_tx_selection_offload = 13;
-	DataPath hints_data_path = 14;
-	bool hints_preemption = 15;
-	bool hints_launch_time_control = 16;
+	bool talker = 11;
```

Comment:
Why is this required if we have separated messages for the QoS requests and responses for talker and listener?